tests/integration/errors/test_E_USER_ERROR.php

```diff
@@ -5,17 +5,7 @@
  */
 
 /*DESCRIPTION
-The agent should capture and report user-generated fatal errors along with a
-stack trace.
-*/
-
-/*INI
-display_errors=1
-log_errors=0
-*/
-
-/*EXPECT_REGEX
-^\s*(PHP )?Fatal error:\s*Sample E_USER_ERROR in .*? on line [0-9]+\s*$
```

Comment:
Why was this removed?

Reply:
Please see: [8c169db](https://github.com/newrelic/newrelic-php-agent/pull/850/commits/8c169db45c7bdb8b26d19dae3cfa28a6d2e40813)